packages/api/test/car.spec.js

```diff
@@ -19,7 +19,7 @@ describe('POST /car', () => {
     const { root, car: carBody } = await createCar('hello world!')
 
     // expected CID for the above data
-    const expectedCid = 'bafybeiczsscdsbs7ffqz55asqdf3smv6klcw3gofszvwlyarci47bgf354'
+    const expectedCid = 'bafkreidvbhs33ighmljlvr7zbv2ywwzcmp5adtf4kqvlly67cy56bdtmve'
```

Comment:
this was getting the empty directory CID because default wrapWithDirectory true and no path. Changed it in the createCar function